templates/org.html

```diff
@@ -92,6 +92,20 @@ <h1 lang="{{ .orgLabel.Lang }}">{{ .orgLabel }}</h1>
                 </ul>
                 
                 {{ if $details.wikipedia }}<p><a href="{{ $details.wikipedia }}">Read more on Wikipedia.</a></p>{{ end }}
+
+                <div class="additional-info collapsed">
+                    <hr>
+                    <ul>
+                        <li>Official jobs page: <span class="text-grey" id="additional-link-jobs">[missing data]</span></li>
+                        <li>Official blog: <span class="text-grey" id="additional-link-blog">[missing data]</span></li>
```

Comment:
Suggestion: omit the `[missing data]` from these 2. Because it is not very common those links and if there is something “missing” **even if there is none** does not come across very well in my opinion.

Reply:
I don't agree. This is an invitation to help add the data. And these are just the first few things we add, just with all open issues there may be 3-4 more.

Reply:
I'm not a huge fan of this pattern either as a user can see the expand button just to get disappointed. However, people looking to contribute might not even know what they can contribute if we do not indicate this, this is for example a known problem with the call to action buttons. For example if you go to an agency missing an email address there is no way to know from that page that adding an email address will add a button.

For the future I imagine these "missing data" sections(both here and in the existing dialog) to be either accompanied with or replaced by "add"/"+" buttons which would open a modal and let people authenticate with OAuth and paste a new link.